Emby.Dlna/Api/DlnaServerService.cs

```diff
@@ -127,11 +127,14 @@ public class DlnaServerService : IService, IRequiresRequest
         public DlnaServerService(
             IDlnaManager dlnaManager,
             IHttpResultFactory httpResultFactory,
-            IServerConfigurationManager configurationManager)
+            IServerConfigurationManager configurationManager,
+            IHttpContextAccessor httpContextAccessor)
         {
             _dlnaManager = dlnaManager;
             _resultFactory = httpResultFactory;
             _configurationManager = configurationManager;
+            object request = httpContextAccessor?.HttpContext.Items["ServiceStackRequest"] ?? throw new ArgumentNullException(nameof(httpContextAccessor));
+            Request = (IRequest)request;
```

Comment:
I think `IRequiresRequest` can be removed from this class now right?

Reply:
You're right - it only set the request variable creation.